config.go

```diff
@@ -105,6 +105,10 @@ const (
 	// initiated the channel closure.
 	defaultCoopCloseTargetConfs = 6
 
+	// defaultAnchorsCommitmentConfTarget is the default confirmation target
+	// used to estimate the fee to use for anchor commitments.
+	defaultAnchorsCommitmentConfTarget = 6
```

Comment:
Does this also imply the Target for the HTLC-Timeout and HTLC-Success Transactions? When Talking about Commitment TX are we always talking about the whole package ?

Reply:
hmmm good question. I think it might apply to the htlc txs as well _but_ when using option_anchors, these htlc txs are signed with `SIGHASH_SINGLE|SIGHASH_ANYONECANPAY` meaning that their fees can easily be adjusted by adding inputs & outputs